2021-03-12 18:32:05,562 INFO    Thread-11 :9152 [wandb_setup.py:_flush():69] setting env: {'project': 'uncategorized', 'entity': 'avyay', 'root_dir': 'C:\\Users\\rao_a\\PythonCPU\\Deep Learning\\CS6910-Assignments-1\\Assignment1\\Avyay', 'sweep_id': '3qzm6h8l', 'run_id': '76gd11qa', 'sweep_param_path': 'C:\\Users\\rao_a\\PythonCPU\\Deep Learning\\CS6910-Assignments-1\\Assignment1\\Avyay\\wandb\\sweep-3qzm6h8l\\config-76gd11qa.yaml'}
2021-03-12 18:32:05,563 INFO    Thread-11 :9152 [wandb_setup.py:_flush():69] setting login settings: {}
2021-03-12 18:32:05,563 INFO    Thread-11 :9152 [wandb_init.py:_log_setup():334] Logging user logs to C:\Users\rao_a\PythonCPU\Deep Learning\CS6910-Assignments-1\Assignment1\Avyay\wandb\run-20210312_183205-76gd11qa\logs\debug.log
2021-03-12 18:32:05,564 INFO    Thread-11 :9152 [wandb_init.py:_log_setup():335] Logging internal logs to C:\Users\rao_a\PythonCPU\Deep Learning\CS6910-Assignments-1\Assignment1\Avyay\wandb\run-20210312_183205-76gd11qa\logs\debug-internal.log
2021-03-12 18:32:05,564 INFO    Thread-11 :9152 [wandb_init.py:init():367] calling init triggers
2021-03-12 18:32:05,565 INFO    Thread-11 :9152 [wandb_init.py:init():374] wandb.init called with sweep_config: {'layer_size': 32, 'n_layers': 3, 'optimizer': 'sgd'}
config: {'n_layers': 3, 'layer_size': 32, 'weight_decay': 0, 'lr': 0.001, 'optimizer': 'sgd', 'batch_size': 32, 'init_method': 'random', 'activation': 'relu', 'epochs': 5}
2021-03-12 18:32:05,565 INFO    Thread-11 :9152 [wandb_init.py:init():389] re-initializing run, found existing run on stack: 76gd11qa
2021-03-12 18:32:05,569 INFO    Thread-11 :9152 [wandb_run.py:finish():1135] finishing run avyay/uncategorized/76gd11qa
2021-03-12 18:32:05,570 INFO    Thread-11 :9152 [jupyter.py:save_history():320] not saving jupyter history
2021-03-12 18:32:05,570 INFO    Thread-11 :9152 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-03-12 18:32:05,570 INFO    Thread-11 :9152 [wandb_init.py:_jupyter_teardown():266] cleaning up jupyter logic
2021-03-12 18:32:05,570 INFO    Thread-11 :9152 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 0
2021-03-12 18:32:05,571 INFO    Thread-11 :9152 [wandb_run.py:_restore():1418] restore
2021-03-12 18:32:07,013 INFO    Thread-11 :9152 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 921
  total_bytes: 921
}

2021-03-12 18:32:09,015 INFO    Thread-11 :9152 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4599
  total_bytes: 4599
}

2021-03-12 18:32:11,017 INFO    Thread-11 :9152 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4599
  total_bytes: 4599
}

2021-03-12 18:32:12,203 INFO    Thread-11 :9152 [wandb_run.py:_show_files():1790] logging synced files
2021-03-12 18:32:12,212 INFO    Thread-11 :9152 [wandb_init.py:init():416] starting backend
2021-03-12 18:32:12,212 INFO    Thread-11 :9152 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=spawn, using: spawn
2021-03-12 18:32:12,216 INFO    Thread-11 :9152 [backend.py:ensure_launched():123] starting backend process...
2021-03-12 18:32:12,313 INFO    Thread-11 :9152 [backend.py:ensure_launched():128] started backend process with pid: 4100
2021-03-12 18:32:12,314 INFO    Thread-11 :9152 [wandb_init.py:init():421] backend started and connected
2021-03-12 18:32:12,316 INFO    Thread-11 :9152 [wandb_run.py:_config_callback():733] config_cb None None {'layer_size': 32, 'n_layers': 3, 'optimizer': 'sgd'}
2021-03-12 18:32:12,316 INFO    Thread-11 :9152 [wandb_init.py:init():461] updated telemetry
2021-03-12 18:32:12,317 INFO    Thread-11 :9152 [wandb_init.py:init():480] communicating current version
2021-03-12 18:32:14,564 INFO    Thread-11 :9152 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.10.22 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-03-12 18:32:14,565 INFO    Thread-11 :9152 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-03-12 18:32:15,224 INFO    Thread-11 :9152 [wandb_init.py:init():518] starting run threads in backend
2021-03-12 18:32:17,014 INFO    Thread-11 :9152 [wandb_run.py:_console_start():1476] atexit reg
2021-03-12 18:32:17,015 INFO    Thread-11 :9152 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.WRAP
2021-03-12 18:32:17,015 INFO    Thread-11 :9152 [wandb_run.py:_redirect():1374] Wrapping output streams.
2021-03-12 18:32:17,015 INFO    Thread-11 :9152 [wandb_run.py:_redirect():1390] Redirects installed.
2021-03-12 18:32:17,015 INFO    Thread-11 :9152 [wandb_init.py:init():542] run started, returning control to user process
2021-03-12 18:32:22,934 INFO    Thread-11 :9152 [wandb_run.py:finish():1135] finishing run avyay/uncategorized/76gd11qa
